src/arr/compiler/anf-loop-compiler.arr

```diff
@@ -101,6 +101,8 @@ j-for = J.j-for
 j-raw-code = J.j-raw-code
 make-label-sequence = J.make-label-sequence
 
+var cases-dispatches = cl-empty
```

Comment:
Could you make it be a box on the compiler object?  We don't really support raw `ref` fields on the compiler object itself, but you could make a small data instance:

```
data Dispatches:
  | dispatches-box(ref dispatches)
end
```

and then construct a new one and put it aside the other accumulators on the compiler like get-loc and current-ans.  It would still be stateful but it would be instantiated per-call to compile-module